build.cmd

```diff
@@ -527,7 +542,7 @@ if defined TF_BUILD (
     git fetch --all
 )
 
-set msbuildflags=/maxcpucount %_nrswitch% /nologo
+set msbuildflags=%_nrswitch% /nologo
```

Comment:
No it wasn't.  I actually submitted a proposal yesterday to remove it.  I had build errors I couldn't understand all weekend.  The internal signed build failed for the last 4 builds.